contracts/src/v0.8/dev/KeeperRegistryBase.sol

```diff
@@ -155,6 +155,7 @@ abstract contract KeeperRegistryBase is ConfirmedOwner, ExecutionPrevention, Ree
   event UpkeepCanceled(uint256 indexed id, uint64 indexed atBlockHeight);
   event UpkeepPaused(uint256 indexed id);
   event UpkeepUnpaused(uint256 indexed id);
+  event UpkeepUpdated(uint256 indexed id);
```

Comment:
nit. Call it UpkeepCheckDataUpdated. And also include bytes checkData in the event?

Reply:
My intuition says that it's better to have more info in events if it's not on a hot path. Off chain listeners can ignore it if they want. Don't feel too strongly about it though, especially since we don't seem to have a use case for it should be ok.

Regarding the event name, at least in the current form I find `UpkeepUpdated` pretty confusing. Gas limit change / pausing / unpausing is also an update.

Reply:
Done